nix/shell.nix

```diff
@@ -63,7 +63,6 @@ craneLib.devShell {
   venvDir = "./.venv";
   postVenvCreation = ''
     unset SOURCE_DATE_EPOCH
-    make generate-python-sdk
```

Comment:
You will need to make sure that `sdk/python` is copied over in the in the flake.nix as well. And that those directories are tracked for changes by nix.

Reply:
How should I proceed to have the sdk/python copied over in the flake.nix ?

Reply:
the smoke test target should include the relative path to the sdk/python to react to changes.

Reply:
https://github.com/hoprnet/hoprnet/blob/b802ea6856638121a6e48c113336e41a87c2e241/flake.nix#L323